adam-cli/src/main/scala/org/bdgenomics/adam/cli/View.scala

```diff
@@ -126,8 +126,8 @@ class View(val args: ViewArgs) extends BDGSparkCommand[ViewArgs] {
       getFilter(0x8, read => read.getReadPaired && !read.getMateMapped),
       getFilter(0x10, _.getReadNegativeStrand),
       getFilter(0x20, _.getMateNegativeStrand),
-      getFilter(0x40, _.getFirstOfPair),
-      getFilter(0x80, _.getSecondOfPair),
+      getFilter(0x40, _.getReadNum == 0),
+      getFilter(0x80, _.getReadNum == 1),
```

Comment:
This isn't related to this PR. I think the name `getReadNum` needs to be clarified. When I first saw it, I thought it was a unique ID per read (and went to look at the source for how we implemented it). Maybe `getFragmentReadNumber` or somesuch name will make it clearer. Nothing to change for this PR.